pkg/virt-controller/services/template.go

```diff
@@ -264,7 +264,7 @@ func (t *templateService) RenderMigrationManifest(vmi *v1.VirtualMachineInstance
 	if namescheme.PodHasOrdinalInterfaceName(network.NonDefaultMultusNetworksIndexedByIfaceName(pod)) {
 		ordinalNameScheme := namescheme.CreateOrdinalNetworkNameScheme(vmi.Spec.Networks)
 		multusNetworksAnnotation, err := network.GenerateMultusCNIAnnotationFromNameScheme(
-			vmi.Namespace, vmi.Spec.Domain.Devices.Interfaces, vmi.Spec.Networks, ordinalNameScheme, t.clusterConfig)
+			vmi.Namespace, vmi.Spec.Domain.Devices.Interfaces, vmi.Spec.Networks, ordinalNameScheme, map[string]network.IPAMClaimParams{}, t.clusterConfig)
```

Comment:
Why are the IPAMClaims removed from the multus annotation on migration?

Reply:
PodHasOrdinalInterfaceName happens only on old scheme for backward compatibility.
Old VMs that ran older versions doesn't support the new feature, so we don't need to add it
(assuming no one adds a network to an old VM)

Reply:
Sumup of what we spoke offline so we can remember it
A VM with old naming will always have old naming
it doesn't support hot unplug but just hotplug.
Since in this PR we don't cover yet IPAM for hotplug / unplug there is no need to support hotplug for old naming scheme as well, so once we do support hotplug it can be revised.

Thanks